docker-compose.yml

```diff
@@ -11,15 +11,27 @@ services:
     ports:
       - "5050:5050"
     depends_on:
-      - "mongo"
-  mongo:
+      - "graphelier-db"
+  graphelier-db:
     build:
       context: ./core/db
     container_name: "graphelier-db"
     ports:
       - "27017:27017"
-    volumes:
-      - "graphelier-db:/data/db"
+  graphelier-news:
+    build:
+      context: ./core/graphelier_news
+    ports:
+      - "8080:8080"
+    depends_on:
+      - "news-db"
+  news-db:
+    build:
+      context: ./core/news-db
+    container_name: "news-db"
+    ports:
+      - "27016:27017"
```

Comment:
port 27017 of the mongo news-db container to public 27016 right ?

Reply:
yep